mbed_build/__main__.py

```diff
@@ -1,8 +0,0 @@
-#
```

Comment:
I kinda liked the fact we could use this for testing it standalone. Granted it'd require wrapping in click group for it work with multiple commands now, but I still think it's useful to have.

Reply:
Ah ok, I was following what we were doing in mbed-project. Will put it back and try and get it working.

Reply:
Personally, I think `mbed-project` is the worst package of the family. It has @rwalton-arm all over it. :trollface:

Reply:
Thanks

Reply:
Hey, no bickering on my pull requests! 😄  I honestly have no particular preference either way. It was just more of a faff to get the click group stuff working.

Reply:
ef9592e